elasticsearch/probe/readiness.sh

```diff
@@ -59,5 +61,19 @@ function show_failures() {
   fi
 }
 
+SECURITY_INDEX=".security"
+docs=(roles rolesmapping actiongroups internalusers config)
+
 check_if_ready "/" "Elasticsearch node is not ready to accept HTTP requests yet"
-check_for_init_complete || ( check_for_init_running && echo "Elasticsearch node is initializing" ) || show_failures
+check_index_exists "${SECURITY_INDEX}"
+
+for d in $docs
+do
+    check_index_exists "${SECURITY_INDEX}/security/${d}"
+done
+
+for template_file in ${ES_HOME}/index_templates/*.json
+do
+    template=`basename $template_file`
+    check_index_exists "_template/${template}"
+done
```

Comment:
So readiness no longer depends on file exists but a exit 0 of this script which is:  security docs exist, templates exist

Reply:
yeah, we were creating files based on if we finished seeding... as long as its available to the cluster we should consider a node ready (and we can get to it)